tests/x11/firefox.pm

```diff
@@ -23,9 +23,10 @@ sub run() {
 
     $self->start_firefox;
     wait_still_screen;
-    send_key_until_needlematch('firefox-top-bar-highlighted', 'alt', 4, 10);
-    send_key('h');
-    wait_still_screen;
+    send_key('alt', 5);
```

Comment:
@tinawang123 could you please add some comments here why you did these changes?

Reply:
I agree, add some comments here for the change to send 'alt' then send 'alt-v' will help others to understand the code.